cmd/agent/app/integrations.go

```diff
@@ -162,6 +112,45 @@ var showCmd = &cobra.Command{
 	RunE:  show,
 }
 
+func getIntegrationName(packageName string) string {
```

Comment:
I remember having to special-case my own code like this, so I understand the feeling. However, this is not going to scale unless we remember to keep updating this hardcoded list? Is there something we can do to fix the problem long term? @ofek

Reply:
Agreed. But I don't think this list should grow that much. Every name for new integrations should be normalized now.

Reply:
ok